server/main.lua

```diff
@@ -50,7 +50,8 @@ lib.callback.register("qb-garage:server:validateGarageVehicle", validateGarageVe
 local function checkOwnership(source, plate, type, house, gang)
     local pData = QBCore.Functions.GetPlayer(source)
     if type == "public" then --Public garages only for player cars
-        return MySQL.query.await('SELECT * FROM player_vehicles WHERE plate = ? AND citizenid = ?', {plate, pData.PlayerData.citizenid})
+         local result = MySQL.query.await('SELECT * FROM player_vehicles WHERE plate = ? AND citizenid = ?', {plate, pData.PlayerData.citizenid})
+         return result[1] or false
```

Comment:
You can minimize this by just sending `result[1]`, if it's nil it would basically act the same as being false